src/utils/internal/parseGraphQLRequest.ts

```diff
@@ -161,7 +161,7 @@ export function parseGraphQLRequest(
       `[MSW] Failed to intercept a GraphQL request to "${request.method} ${requestPublicUrl}": cannot parse query. See the error message from the parser below.`,
     )
     console.error(parsedResult)
-    return null
+    return undefined
```

Comment:
Returning `undefined` from the `parseGraphQLRequest` function so its return type is compatible with the `ParsedGraphQLRequest` type. No need to keep `Parsed | null | undefined`, that's confusing. We're eliminating `null` by replacing it with `undefined` here.